airflow/utils/db.py

```diff
@@ -91,6 +91,8 @@
     "2.8.0": "10b52ebd31f7",
     "2.8.1": "88344c1d9134",
     "2.9.0": "1949afb29106",
+    "2.9.1": "bff083ad727d",
+    "2.10.0": "677fdbb7fc54",
```

Comment:
Not sure we should be adding these.
```suggestion
```

Reply:
Even if I remove it, the pre-commit hook check-revision-heads-map adds it back. 

Those references get used when running the commands
`airflow db downgrade --to-version 2.9.1`

`airflow db migrate --to-version 2.9.1`

Reply:
I see the issue here :) Not going to worry about it in this PR though.

Reply:
cc @ephraimbuddy we will want to remove 2.10.0 when we cherry-pick this back to v2-9.